src/validation/ListKnownPartiesResponseValidation.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {CreatedEventValidation} from "./CreatedEventValidation";
+import {ListKnownPartiesResponse} from "../model/ListKnownPartiesResponse";
+import {OptionalFieldsValidators, RequiredFieldsValidators} from "./Validation";
+import {array} from "./Array";
+import {native} from "./Native";
+import {object} from "./Object";
+
+function required(): RequiredFieldsValidators<ListKnownPartiesResponse> {
+    return {
+        parties: array()
```

Comment:
@stefanobaghino-da These are dummy codes.

How do I implement validation of data fields that are of an array type?

Reply:
`array` takes a validation and applies it to an array. Pass a `native('string')` here.